tutorials/desktop/firefox-for-web-developers/firefox-for-web-developers.md

```diff
@@ -0,0 +1,309 @@
+### Using the context menu ###
+There is a third way to open the developer tools. This is not a toggle, as it always switches to a particular tab, but is a very common approach when you want to take a closer look at a specific element on the web page. If you right-click on something in the page to open the context menu, you'll find an entry labelled **Inspect Element** towards the bottom. You can select this to open the developer tools with the **Inspector** tab active and the element you clicked on already highlighted within the panel. Let's give it a try:
+
+1. Right click on the "Do more with Ubuntu!" heading near the top of the web page.
+2. Select **Inspect Element** from the context menu.
+3. The developer tools should open with the Inspector tab selected and the `<h1>` element highlighted.
+
+![Inspecting the page heading](images/inspector.png)
+
+As you can see, this method cuts right through the layers of HTML content to highlight the specific element that you right-clicked on, making this a very convenient way to open the tools if you need to work on a particular part of the page.
+
+
+## The inspector##
+Duration: 8:00
+
+All the tabs in the developer tools panel have their uses, but for beginners to web development — or anyone who is simply curious about how a web page works — you only really need to use the **Inspector** and the **Console**. This section and the next will describe them in a little more detail.
+
+The inspector lets you look at the details of the web page's "Document Object Model" (DOM). This is the browser's internal version of the page structure, and the inspector exposes it in a way that makes it look similar to the HTML that was used to create the page. Be clear, though, that although the DOM is initially created from the HTML, it's not quite the same thing. The DOM is a live representation of the page structure, so if the page content has been dynamically modified by some JavaScript code the changes will be reflected in the DOM, and hence in the inspector. The live nature of the DOM also means that you can change things in the inspector and see the results in real-time on the page — ideal for interactively tweaking the design of your page.
+
+Let's play around with the Ubuntu Tutorials page a little. Don't worry, nothing you do in the inspector can be saved back to the Ubuntu servers, so your changes will only last until you reload the page and won't affect anyone else. The same applies to any other page — the developer tools let you tweak your page interactively, but those changes won't stick unless they're also applied to the original source code on the server.
+
+Begin by right-clicking on the "Do more with Ubuntu!" heading and selecting **Inspect Element** from the context menu to highlight the `<h1>` element in the inspector (you might already be in this position from the previous section, but it won't hurt to do it again to be sure). As a reminder, here's how the developer tools should look at this point:
```

Comment:
I think it's relatively likely that the title of the homepage might change before too long, and the chance of anyone remembering to update this tutorial is practically zero. Would it be possible to simply refer to it as the "page title" or something a little less specific? Maybe you could mention that it's "Do more with Ubuntu!" at the time of writing, but might have changed?